zetaclient/chains/solana/signer/signer.go

```diff
@@ -299,8 +315,21 @@ func (signer *Signer) broadcastOutbound(
 				logger.Error().Err(err).Fields(lf).Msgf("unable to deserialize PDA info")
 			} else if pda.Nonce > nonce {
 				logger.Info().Err(err).Fields(lf).Msgf("PDA nonce %d is greater than outbound nonce, stop retrying", pda.Nonce)
+				if nonce > signer.maxProcessedNonce.Load() {
```

Comment:
The name `maxProcessedNonce` is unclear to me. Does it mean the `max nonce processed only by me` or `the gateway pending nonce`, similar to Ethereum pending nonce. I think giving a consistent name helps understanding, Bitcoin chain also maintains its own pending nonce in zetaclient.

Reply:
> max nonce processed only by me

yes that is want I mean

I should call it `maxRelayedNonce`